ui/perfherder/perf-helpers/constants.js

```diff
@@ -204,3 +204,94 @@ export const noiseProfiles = {
+];
+
+/**
+ * Used for building the documentation links (Perfdocs) and side-by-side links.
+ * @link https://firefox-source-docs.mozilla.org/testing/perfdocs/raptor.html#benchmarks
+ */
+export const browsertimeBenchmarksTests = [
+  'ares6',
+  'assorted-dom',
+  'jetstream2',
+  'matrix-react-bench',
+  'motionmark-animometer',
+  'motionmark-htmlsuite',
+  'raptor-speedometer-geckoview',
+  'raptor-youtube-playback-av1-sfr-chrome',
+  'raptor-youtube-playback-av1-sfr-fenix',
+  'raptor-youtube-playback-av1-sfr-firefox',
+  'raptor-youtube-playback-av1-sfr-geckoview',
+  'raptor-youtube-playback-h264-1080p30-firefox',
+  'raptor-youtube-playback-h264-1080p60-firefox',
+  'raptor-youtube-playback-h264-full-1080p30-firefox',
+  'raptor-youtube-playback-h264-full-1080p60-firefox',
+  'raptor-youtube-playback-h264-sfr-chrome',
+  'raptor-youtube-playback-h264-sfr-fenix',
+  'raptor-youtube-playback-h264-sfr-firefox',
+  'raptor-youtube-playback-h264-sfr-geckoview',
+  'raptor-youtube-playback-hfr-chrome',
+  'raptor-youtube-playback-hfr-fenix',
+  'raptor-youtube-playback-hfr-firefox',
+  'raptor-youtube-playback-hfr-geckoview',
+  'raptor-youtube-playback-v9-1080p30-firefox',
+  'raptor-youtube-playback-v9-1080p60-firefox',
+  'raptor-youtube-playback-v9-full-1080p30-firefox',
+  'raptor-youtube-playback-v9-full-1080p60-firefox',
+  'raptor-youtube-playback-vp9-sfr-chrome',
+  'raptor-youtube-playback-vp9-sfr-fenix',
+  'raptor-youtube-playback-vp9-sfr-firefox',
+  'raptor-youtube-playback-vp9-sfr-geckoview',
+  'raptor-youtube-playback-widevine-h264-sfr-chrome',
+  'raptor-youtube-playback-widevine-h264-sfr-fenix',
+  'raptor-youtube-playback-widevine-h264-sfr-firefox',
+  'raptor-youtube-playback-widevine-h264-sfr-geckoview',
+  'raptor-youtube-playback-widevine-hfr-chrome',
+  'raptor-youtube-playback-widevine-hfr-fenix',
+  'raptor-youtube-playback-widevine-hfr-firefox',
+  'raptor-youtube-playback-widevine-hfr-geckoview',
+  'raptor-youtube-playback-widevine-vp9-sfr-chrome',
+  'raptor-youtube-playback-widevine-vp9-sfr-fenix',
+  'raptor-youtube-playback-widevine-vp9-sfr-firefox',
+  'raptor-youtube-playback-widevine-vp9-sfr-geckoview',
+  'speedometer',
+  'stylebench',
+  'sunspider',
+  'unity-webgl',
+  'wasm-godot',
+  'wasm-godot-baseline',
+  'wasm-godot-optimizing',
+  'wasm-misc',
+  'wasm-misc-baseline',
+  'wasm-misc-optimizing',
+  'webaudio',
+  'youtube-playback',
+  'youtube-playback-av1-sfr',
+  'youtube-playback-h264-1080p30',
+  'youtube-playback-h264-1080p60',
+  'youtube-playback-h264-full-1080p30',
+  'youtube-playback-h264-full-1080p60',
+  'youtube-playback-h264-sfr',
+  'youtube-playback-hfr',
+  'youtube-playback-v9-1080p30',
+  'youtube-playback-v9-1080p60',
+  'youtube-playback-v9-full-1080p30',
+  'youtube-playback-v9-full-1080p60',
+  'youtube-playback-vp9-sfr',
+  'youtube-playback-widevine-h264-sfr',
+  'youtube-playback-widevine-hfr',
+  'youtube-playback-widevine-hfr',
+];
+
+export const browsertime = 13;
```

Comment:
```suggestion
export const browsertimeId = 13;
```
Or a similar name that would suggest this constant is actually the id or pk for the framework browsertime.

Reply:
the original intent was to have something like `frameworks.browsertime` for readability